mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -118,6 +118,14 @@ pub enum MakerSwapEvent {
         maker_payment: TransactionIdentifier,
         taker_payment: TransactionIdentifier,
     },
+    /// `Taker payment` has been received and preimage of `taker payment spend` has been skipped.
```

Comment:
`` used for name ids mostly, you should prefer '' for these.

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2359/commits/61d0960a84b529d680ae844e822930a8f29a47e2